src/migrations/r-migration-commands/m130_131_add_names_for_unnamed_rules.ts

```diff
@@ -23,7 +23,7 @@ class Migration implements IOrder {
       }
     }
 
-    projectF.set(rtp.project.version, 130);
+    projectF.set(rtp.project.version, 131);
```

Comment:
is this intentional? Changing this can impact existing projects that have already gone through this migration.

Reply:
Yes, intentional change. 130 was present in two migrations.